examples/src/migration/README.rst

```diff
@@ -0,0 +1,22 @@
+##################
+Migration Examples
+##################
+
+The `Encryption SDK for Python`_ now uses the `AWS Cryptographic Material Providers Library`_. The MPL abstracts lower
+level cryptographic materials management of encryption and decryption materials.
+
+This directory contains migration examples for:
+
+* Moving to Keyrings from Master Key Providers:
+    #. Migration example to AWS KMS keyring from AWS KMS Master Key Provider.
+    #. Migration example to Raw AES keyring from Raw AES Master Key Provider.
+    #. Migration example to Raw RSA keyring from Raw RSA Master Key Provider.
+    
+* Migration to newer versions of the ESDK from the old version (1.x):
```

Comment:
```suggestion
* Migration to newer versions of the ESDK (4.x+) from 1.x versions:
```
This is pretty specific to 1.x, let's leave out any references to "old versions" since 1.x covers the scope of impacted versions

Reply:
Okay